webapps/src/main/runtime/test/java/org/camunda/bpm/pa/rest/TestServlet.java

```diff
@@ -153,7 +153,13 @@ protected void createAdminUser(ProcessEngine processEngine) {
     userProfileDto.setLastName("Hentschi");
     user.setProfile(userProfileDto);
 
-    new SetupResource().createInitialUser(processEngine.getName(), user);
+    try {
+      new SetupResource().createInitialUser(processEngine.getName(), user);
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
I guess in the `TestServlet` it doesn't matter whether we use logger or just plain console print but other parts of the class use java logger.

Reply:
Yeah, I agree that it doesn't really matter since it only concerns our development build. I adjusted it to wrap the exception in a `RuntimeException` so that the compiler warning disappears.